platooning_strategic_IHP/include/platoon_manager_ihp.h

```diff
@@ -245,6 +253,18 @@ namespace platoon_strategic_ihp
         */
         double getIHPDesPosFollower(double dt);
 
+        /**
+        \brief UCLA: Return joiner's desired position
+        to cut into the platoon 
+        */
+        int getClosestIndex(double joinerDtD);
+
+        /**
+        \brief UCLA: Return the desired gap size
```

Comment:
Units of measure?
Need descriptions of both parameters.  don't assume I know what "leading" means :)